data/datasets/semantics_ws_qna_oa/random_stuff.py

```diff
@@ -0,0 +1,249 @@
+]
+
+# 50%;
+random_list_rel_en_a_50 = [
+    "There is a slight association between the words {word1} and {word2}.",
+    'There is an indirect semantic similarity between the words "{word1}" and "{word2}".',
+    "There is some association between the words '{word1}' and '{word2}'.",
+]
+random_list_rel_ru_a_50 = [
+    "Есть небольшая ассоциация между словами {word1} и {word2}.",
+    'Между словами "{word1}" и "{word2}" есть непрямое семантическое сходство.',
+    "Есть некоторая ассоциативность между словами '{word1}' и '{word2}'.",
+]
+random_list_rel_de_a_50 = [
+    "Es besteht eine leichte Assoziation zwischen den Wörtern {word1} und {word2}.",
+    'Es gibt eine indirekte semantische Ähnlichkeit zwischen den Wörtern "{word1}" und "{word2}".',
+    "Es besteht eine gewisse Verbindung zwischen den Wörtern '{word1}' und '{word2}'.",
+]
+random_list_rel_it_a_50 = [
+    "C'è una leggera associazione tra le parole {word1} e {word2}.",
+    'C\'è una somiglianza semantica indiretta tra le parole "{word1}" e "{word2}".',
+    "C'è una certa associazione tra le parole '{word1}' e '{word2}'.",
+]
+
+# no;
+random_list_rel_en_a_n = [
+    "No, there is no associative relationship between the words {word1} and {word2}",
+    'No, the words "{word1}" and "{word2}" are not related in any direct associative way.',
+    "There is no direct associative relationship between '{word1}' and '{word2}'.",
+]
+random_list_rel_ru_a_n = [
+    "Нет, между словами {word1} и {word2} нет ассоциативной связи",
+    'Нет, слова "{word1}" и "{word2}" не связаны каким-либо прямым ассоциативным образом.',
+    "Нет никакой прямой ассоциативной связи между '{word1}' и '{word2}'.",
+]
+random_list_rel_de_a_n = [
+    "Nein, es besteht keine Assoziationsbeziehung zwischen den Wörtern {word1} und {word2}.",
+    'Nein, die Wörter "{word1}" und "{word2}" sind nicht in direkter assoziativer Weise miteinander verbunden.',
+    "Es besteht keine direkte assoziative Beziehung zwischen '{word1}' und '{word2}'.",
+]
+random_list_rel_it_a_n = [
+    "No, non c'è alcuna relazione associativa tra le parole {word1} e {word2}.",
+    'No, le parole "{word1}" e "{word2}" non sono correlate in alcun modo associativo diretto.',
+    "Non c'è una diretta relazione associativa tra '{word1}' e '{word2}'.",
+]
+
+# easy-way to call stuff above
+
+# dicts of q
+# sim
+random_dict_sim_q = {
+    "en": random_list_sim_en_q,
+    "ru": random_list_sim_ru_q,
+    "de": random_list_sim_de_q,
+    "it": random_list_sim_it_q,
+}
+# rel
+random_dict_rel_q = {
+    "en": random_list_rel_en_q,
+    "ru": random_list_rel_ru_q,
+    "de": random_list_rel_de_q,
+    "it": random_list_rel_it_q,
+}
+
+# dicts for a
+# sim - random_dict_sim_a["ru"][0]  # returns the list of "yes" answers for Russian
+random_dict_sim_a = {
+    "en": {
+        0: random_list_sim_en_a_y,
+        1: random_list_sim_en_a_50,
+        2: random_list_sim_en_a_n,
```

Comment:
here you can also use lists or maybe np.random.choice([elem1, elem2]) instead of elem[randint] but it's ok